### DISCUSSION TITLE
Can a SSH node join two distinct clusters?

### DISCUSSION THREAD

**Discussion by xrisk** [gravitational/teleport#30210] [Q&A] [answered]
I’m setting up a poor man’s version of HA due to infrastructure constraints. To do this, I’ve setup two separate clusters on machine A and B. I then  joined both clusters A and B from a third machine C. I’ve separated the teleport.yaml files and the data directories.

So far, it seems to be working as expected — is this setup fine or will things crash and burn somewhere down the line?

**Comment by webvictim** [comment, marked as answer]
Yes, this is fine and it's how we've recommended others do it in the past. As long as you have separate config files and data directories (and probably `systemd` units for management), it will work.

One thing that's worth bearing in mind is that when you upgrade the `teleport` binary on the machine, it will change the version connected to both clusters. If you want to keep separate binaries at different versions under separate paths, that's also an option.

**Reply by xrisk** [reply]
Thank you!
